src/debugger/adapter/DebugAdapter.js

```diff
@@ -99,12 +98,18 @@ class PrepackDebugSession extends LoggingDebugSession {
 
   // Override
   launchRequest(response: DebugProtocol.LaunchResponse, args: LaunchRequestArguments): void {
+    let inFilePath = path.join(__dirname, "../common/", args.debugInFilePath);
```

Comment:
Any reason debugInFilePath is a relative path? I would assume client pass absolute path.

Reply:
I will change to autogenerate a temporary file in the next PR.